lib/uppy/ping/scheme.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Uppy.Ping do
```

Comment:
Is there a typo in this filename? `scheme`/`schema`

Reply:
[4f26776](https://github.com/andreicek/uppy/pull/2/commits/4f26776e08e2c02f021be8ec320c09cc18d0ac88)